docs/provisioner/08-02-provisioning-gardener.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Provision clusters on GCP, AWS, or AZURE through Gardener
+type: Tutorials
+---
+
+This tutorial shows how to provision clusters with Kyma Runtimes (Runtimes) on Google Cloud Platform (GCP), Microsoft Azure, and Amazon Web Services (AWS) using [Gardener](https://dashboard.garden.canary.k8s.ondemand.com).
+
+## Prerequisites
+
+<div tabs name="Prerequisites" group="Provisioning-Gardener">
+  <details>
+  <summary>
```

Comment:
I think you should add `label` as well here as you have 2 toggles in the doc so they make a set -> [link](https://kyma-project.io/community/guidelines/content/#toggle-toggle-writing-rules)